src/main/java/Task.java

```diff
@@ -0,0 +1,19 @@
+// Add the ability to store whatever text entered by the user and display them back to the user when requested.
+public class Task {
+    
+
+    String[] taskList = new String[100];
+    int taskCount = 0;
```

Comment:
Variable names should be plural


```suggestion
    String[] tasksList = new String[100];
    int tasksCount = 0;
```